Makefile

```diff
@@ -202,7 +202,12 @@ test-integration:
 
 
 # Intended for CI, assumed to be running in quay.io/libpod/skopeo_cidev container.
+ifdef SKOPEO_BINARY
+$(info Skipping build as SKOPEO_BINARY is specified)
+test-integration-local:
+else
 test-integration-local: bin/skopeo
+endif
```

Comment:
This suggests that `SKOPEO_BINARY` might be set. (I can’t see anything setting it. for integration tests.)

But the Go test code invoked by `go test` just assumes Skopeo is found in `PATH`, and noting sets up `SKOPEO_BINARY` to be handled.

How is this intended to work?